### ISSUE TITLE
In order to achieve the same effect, the adjustment of learning rate and batch size

### ISSUE THREAD

**Issue by yaohusama** [open-mmlab/mmsegmentation#2771]
When reducing the batch size to half of the original, can the same effect be achieved by reducing the learning rate to half of the original？I searched all the issues, but couldn't find the answer to the related question。

**Comment by yaohusama** [comment]
Below is mmdetection's webpage explaining this problem.https://github.com/open-mmlab/mmdetection/issues/6263

> From the page [mmdetection -Train predefined models on standard datasets](https://mmdetection.readthedocs.io/en/latest/1_exist_data_model.html#train-predefined-models-on-standard-datasets)

Important: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8*2 = 16). According to the linear scaling rule, you need to set the learning rate proportional to the batch size if you use different GPUs or images per GPU, e.g., lr=0.01 for 4 GPUs * 2 imgs/gpu and lr=0.08 for 16 GPUs * 4 imgs/gpu.

If I have a single GPU and a batch size of 16, I go with lr=0.02?

**Comment by yaohusama** [comment]
If I reduce the batch size, do I need to increase the number of iterations at the same time to achieve the same effect?

**Comment by xiexinch** [comment]
It is true that the number of iterations needs to be increased, but there is no guarantee that the same results will be achieved.

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
